extract/__tests__/fixtures/toc-json.es6

```diff
@@ -1 +1,134 @@
-export default {"byId":{"1":{"childNodes":[],"id":"1","isLeaf":true,"href":"s001-BookTitlePage-01.xhtml","label":"A Christmas Carol","linear":null,"manifestId":"s001","parentId":"__root__"},"2":{"childNodes":[],"id":"2","isLeaf":true,"href":"s002-Copyright-01.xhtml","label":"Copyright","linear":null,"manifestId":"s002","parentId":"__root__"},"3":{"childNodes":[],"id":"3","isLeaf":true,"href":"s003-Introduction-01.xhtml","label":"Introduction","linear":null,"manifestId":"s003","parentId":"__root__"},"4":{"childNodes":[],"id":"4","isLeaf":true,"href":"s004-Preface-01.xhtml","label":"Dramatis Personae","linear":null,"manifestId":"s004","parentId":"__root__"},"5":{"childNodes":[],"id":"5","isLeaf":true,"href":"s005-AboutTheAuthor-01.xhtml","label":"Preface","linear":null,"manifestId":"s005","parentId":"__root__"},"6":{"childNodes":[],"id":"6","isLeaf":true,"href":"s006-Chapter-001.xhtml","label":"Stave 1: Marley’s Ghost","linear":null,"manifestId":"s006","parentId":"__root__"},"7":{"childNodes":[],"id":"7","isLeaf":true,"href":"s007-Chapter-002.xhtml","label":"Stave 2: The First of the Three Spirits","linear":null,"manifestId":"s007","parentId":"__root__"},"8":{"childNodes":[],"id":"8","isLeaf":true,"href":"s008-Chapter-003.xhtml","label":"Stave 3: The Second of the Three Spirits","linear":null,"manifestId":"s008","parentId":"__root__"},"9":{"childNodes":[],"id":"9","isLeaf":true,"href":"s009-Chapter-004.xhtml","label":"Stave 4: The Last of the Spirits","linear":null,"manifestId":"s009","parentId":"__root__"},"10":{"childNodes":[],"id":"10","isLeaf":true,"href":"s010-Chapter-005.xhtml","label":"Stave 5: The End of It","linear":null,"manifestId":"s010","parentId":"__root__"},"__root__":{"childNodes":["1","2","3","4","5","6","7","8","9","10"],"id":"__root__","isLeaf":false,"linear":null}},"byManifestId":{"s001":"1","s002":"2","s003":"3","s004":"4","s005":"5","s006":"6","s007":"7","s008":"8","s009":"9","s010":"10"},"items":["1","2","3","4","5","6","7","8","9","10"]}
+export default {
```

Comment:
@eugenserbanescu would you mind reworking the indentation to two spaces per tab?